tick/linear_model/model_linreg.py

```diff
@@ -115,7 +118,7 @@ def _grad(self, coeffs: np.ndarray, out: np.ndarray) -> None:
 
     def _loss(self, coeffs: np.ndarray) -> float:
         if self.dtype is not "float64" and coeffs.dtype is np.float64:
```

Comment:
Maybe this check should be more generic, i.e.
`if self.dtype != coeffs.dtype.name` ?